packages/utils/src/lib/reports/__snapshots__/report-no-categories.md

```diff
@@ -375,20 +425,21 @@ Cumulative Layout Shift measures the movement of visible elements within the vie
 Sum of all time periods between FCP and Time to Interactive, when task length exceeded 50ms, expressed in milliseconds. [📖 Docs](https://developer.chrome.com/docs/lighthouse/performance/lighthouse-total-blocking-time/)
 
 
-
 ## About
 
-Report was created by [Code PushUp](https://github.com/flowup/quality-metrics-cli#readme) on Wed, Sep 15, 2021, 12:00 AM UTC.
+Report was created by [Code PushUp](https://github.com/code-pushup/cli#readme) on Wed, Oct 18, 2023, 7:49 AM UTC.
+
+### Report overview:
 
 |Commit|Version|Duration|Plugins|Categories|Audits|
 |:--|:--:|:--:|:--:|:--:|:--:|
 |Minor fixes (abcdef0123456789abcdef0123456789abcdef01)|`0.0.1`|1.65 s|2|0|52|
 
-The following plugins were run:
+### Plugins overview:
```

Comment:
I don't think these new headings are necessary or particularly descriptive. 
- How is that tiny table a "Report overview"? The report overview is the whole Markdown document, not some minor stats.
- If you insist on having a heading for the plugins table, just use `### Plugins`. Non of our other headings end in a `:` and the word "overview" adds no information.

Reply:
I removed both headings. You are right, The additional text is not that much more informative.

Reply:
reverted it